src/data/compressedlinestorage.cpp

```diff
@@ -19,17 +19,12 @@
 
 #include <cassert>
 #include <cstdlib>
+#include <QtEndian>
 
 #include "utils.h"
 
 #include "data/compressedlinestorage.h"
 
-#ifdef HAVE_HTONS
-#  include <arpa/inet.h>
-#else
-#  define htons(a) glogg_htons(a)
```

Comment:
Produced and error on Mac build:
src/data/compressedlinestorage.cpp:30:11: error: 'htons' macro redefined [-Werror,-Wmacro-redefined]

I decided to switch to Qt version and let it deal with all platforms